framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -415,6 +415,8 @@ private void parseStubUnit(InputStream inputStream) {
                             "No supported annotations found! This likely means stub file %s doesn't import them correctly.",
                             filename));
         }
+        findPackage("java.lang");
```

Comment:
@smillst Is this call necessary? It appears again as an argument on the line below.
If it's needed, a comment would prevent confusion.

Reply:
https://github.com/typetools/checker-framework/commit/a37fdbe51c87f48ede8c119f1c9c23db508e7628